absl/debugging/symbolize_test.cc

```diff
@@ -448,44 +448,74 @@ void ABSL_ATTRIBUTE_NOINLINE TestWithReturnAddress() {
   std::cout << "TestWithReturnAddress passed" << std::endl;
 #endif
 }
+#elif defined(_WIN32)
 
-#elif defined(_WIN32) && defined(_DEBUG)
+// Functions to symbolize. Use C linkage to avoid mangled names.
+extern "C" {
+
+__declspec(noinline) void win32_nonstatic_func() {
+  volatile int x = __LINE__;
```

Comment:
We were wondering if `ABSL_BLOCK_TAIL_CALL_OPTIMIZATION()` is sufficient? What do the extra two lines do?

Reply:
The two lines are to prevent compiler from folding those functions into one, which I suppose is the original purpose of `ABSL_BLOCK_TAIL_CALL_OPTIMIZATION` as well, but MSVC still do function folding if only using `ABSL_BLOCK_TAIL_CALL_OPTIMIZATION` (tested when comment out those two lines).